docs/Discussions

```diff
@@ -0,0 +1,12 @@
+## DevOps Mutualization SIG Discussions
+
+DevOps Mutualization SIG uses Github Discussions that are only visible to members who are part of the DevOps SIG [team](https://github.com/orgs/finos/teams/devops-mutualization-participants/) as agreed upon in the [meeting on Feb 18th 2020](https://github.com/finos/devops-mutualization/issues/18)
+Discussions augments  Issues in this repository for public discussions. 
+
+## How to Participate in DevOps SIG Discussions
+- If you are not in the FINOS Github org, email help@finos.org and request that your Github account is added.
+- Email Amol Shukla or Anders Wallgren so they can add you to the Github team for the DevOps SIG. 
+- You can now view or see Discussions
+
+
```

Comment:
**Please add** : `In order to collaborate in FINOS Team Discussions, you must have a [GitHub Account](https://github.com/) and have signed a [FINOS Contributor License Agreement (CLA)](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Contribution+Compliance+Requirements#ContributionComplianceRequirements-ContributorLicenseAgreement). Please email help@finos.org for help and more information.`